torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1979,7 +1954,24 @@ void ONNXShapeTypeInference(
     for (auto output : clone_node->outputs()) {
       n_graph->registerOutput(output);
     }
-
+    for (size_t i = 0; i < clone_node->outputs().size(); ++i) {
+      torch_to_onnx_output_name[n->output(i)->debugName()] = clone_node->output(i)->debugName();
+    }
+    // Make inferred_shape_data use name from ONNX graph instead of original PyTorch graph
```

Comment:
Would the following be more accurate?
"ONNX graph" -> "temporal ONNX graph for shape inference"?
"PyTorch graph" -> "original ONNX graph"?